translations/base-kor.yaml

```
@@ -74,8 +75,8 @@ mainMenu:
     puzzleDlcViewNow: DLC 보러 가기
     mods:
         title: Active Mods
-        warningPuzzleDLC: Playing the Puzzle DLC is not possible with mods. Please
-            disable all mods to play the DLC.
+        warningPuzzleDLC: 퍼즐 DLC는 모드와 호환되지 않습니다. DLC 플레이 전 모든 모드를 비활성화해 주세요.
```

```suggestion
        warningPuzzleDLC: 퍼즐 DLC는 모드와 호환되지 않습니다. DLC 플레이 전
```

아랫부분에 있는 내용까지 같이 적은건데 깃헙 UI에서 반영하니 이부분이 적용되지 않네요.